tests/chmode1.c

```diff
@@ -66,6 +66,41 @@ test_chmode_parse(void)
 	remove_hook("get_channel_access", chmode_access_hook);
 }
 
+void
+test_chmode_limits(void)
+{
+	char chmode_buf[2 + MAXMODEPARAMS + 1] = "+";
```

Comment:
Why is this 2 + ... instead of 1 + ... ?

Reply:
The string is `+{'l' * (MAXMODEPARAMS+1)}\0` and I thought `1 + 1` looked silly

Reply:
Aha, missed the for loop after was using `MAXMODEPARAMS + 1`